### PR TITLE
feat[WIP]: Single video page

### PR DESCRIPTION
**Pull request by saadahmadhere**
This PR includes:
- Adding single video page (static version).
- Linking the single video page from the card.
- Add static notes functionality to the single video page.

### CODE REVIEW

**Review by omjadhav85 — APPROVED** [review]
UI looks good! Like that you have implemented responsive design for notes section as well. My only suggestion would be to give a darker background to notes to go along with the whole dark theme. Else its all good to merge...great work!

File: src/screens/SingleVideoPage/SingleVideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          title="YouTube video player"
+          frameBorder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowFullScreen
+          loading="lazy"
+          className="video_player"></iframe>
+        <div className="video_actions flex">
+          {/* <button className="btn">
+            <i className="fas fa-thumbs-up fa-lg mr_1"></i>Like
+          </button> */}
```
Comment (abhij1607) [inline src/screens/SingleVideoPage/SingleVideoPage.jsx:23]: remove this, if not required

**Review by abhij1607 — COMMENTED** [review]
Good folder structure
Neat & clean code

Can be merged

File: src/screens/SingleVideoPage/SingleVideoPage.jsx (hunk 1 above)
Comment (saadahmadhere) [reply src/screens/SingleVideoPage/SingleVideoPage.jsx:23]: I may need it for future reference.

**Pull request merged**